python/src/equistore/block.py

```diff
@@ -229,6 +279,34 @@ def add_gradient(
         """
         Add a set of gradients with respect to ``parameters`` in this block.
 
+        Here is an example of how to use this method:
+        >>> import numpy as np
+        >>> block_1 = TensorBlock(
+        ...             values=np.full((3, 1, 1), 1.0),
+        ...             samples=Labels(
+        ...                 ["samples"], np.array([[0], [2], [4]], dtype=np.int32)
+        ...             ),
+        ...             components=[Labels(
+        ...                 ["components"], np.array([[0]], dtype=np.int32))
+        ...             ],
+        ...             properties=Labels(
+        ...                 ["properties"], np.array([[0]], dtype=np.int32)),
+        ...         )
+        >>> block_1.add_gradient(
+        ...     "position",
+        ...     samples=Labels(
+        ...        ["sample", "position"], np.array([[0, -2], [2, 3]], dtype=np.int32)
+        ...     ),
+        ...     data=np.full((2, 1, 1), 11.0),
+        ...     components=[Labels(["components"], np.array([[0]], dtype=np.int32))],
+        ... )
+        >>> print(block_1)
+        TensorBlock
+            samples (3): ['samples']
+            components (1): ['components']
+            properties (1): ['properties']
+            gradients: ['position']
```

Comment:
Regarding the documentation tests error, I don't know why, but that worked for me locally. I think the doctest may not end with an indentation.
```suggestion
        TensorBlock
        samples (3): ['samples']
        components (1): ['components']
        properties (1): ['properties']
        gradients: ['position']
```

Reply:
this breaks however the doctests (which i did not check properly). i ll check for a different solution